good_egg/models.py

```diff
@@ -1,3 +1,40 @@
 from django.db import models
 
 # Create your models here.
+class Incident(models.Model):
+  YES = 'y'
+  NO = 'n'
+  yes_or_no = (
+    (YES, 'yes'),
+    (NO, 'no'),
+  )
+  category = models.CharField(max_length=100)
+  officers =  models.ForeignKey(Officer, on_delete=models.CASCADE, related_name='officer')
+  date = models.CharField(max_length=100)
+  time = models.CharField(max_length=100)
+  location = models.CharField(max_length=100)
+  formal_complaint = models.CharField(max_length=100)
+  formal_complaint_number = models.CharField(max_length=100, choices=yes_or_no, default=NO)
```

Comment:
Should this have `choices=yes_or_no`?

Reply:
No i guess not.  I think default=N/A works just as well